docs/layouts/shortcodes/generated/core_configuration.html

```diff
@@ -423,6 +423,12 @@
             <td>Double</td>
             <td>Define the default false positive probability for lookup cache bloom filters.</td>
         </tr>
+        <tr>
+            <td><h5>lookup.cache.high-priority-pool-ratio</h5></td>
+            <td style="word-wrap: break-word;">0.25</td>
```

Comment:
What is the source of this default value? 0.1 or 0.25?

Reply:
if 0.1, the size is 25MB, 0.25 is 64MB. When I test, the bloom filter may occupy 5-10MB. So, I give a bigger (64MB) default index cache here.